src/styles/index.scss

```diff
@@ -1,3 +1,6 @@
 body {
   margin: 0;
+  background-color: antiquewhite;
```

Comment:
Consider using a variable for colors in SCSS for better maintainability and reusability. For example: `$primary-background: antiquewhite;` and then use `background-color: $primary-background;`.